chapters/connectors.tex

```diff
@@ -903,7 +905,19 @@ \subsection{Connection Graphs and Their Operators}\label{overconstrained-equatio
 
 \subsection{Generation of Connection Graph Equations}\label{generation-of-connection-graph-equations}
 
-Before connection equations are generated, the virtual connection graph is transformed into a set of spanning trees by removing optional spanning tree edges from the graph.
+Only the overdetermined connector components are considered.
+This generation replaces the equality-equation for variables that are neither flow nor stream in \cref{generation-of-connection-equations}.
+
+In order to handle any redundant connect-equation the following preparation is needed.
+\begin{enumerate}
+\item
+  We build the connection sets in the normal way, keeping the overdetermined components as primitives.
```

Comment:
What about overdetermined record instances? They would not qualify as primitives by the definition in 9.2, but unless I'm missing something it seems like they need to be treated as such when building the virtual connection graph (or even the connection sets?).

Reply:
Yes, the idea is that they are included as primitives here. We might clarify that further.